src/accountsMetadata.ts

```diff
@@ -0,0 +1,106 @@
+import { OutputVendorName } from '@/originalBudgetTrackingApp/configManager/configManager';
+import { CompanyTypes } from 'israeli-bank-scrapers-core';
```

Comment:
I'm thinking about the `israeli-bank-scrapers` updates. If we want to derived their list to attach some custom data, I think it will be good if we validate that we calling the `CompanyTypes` from the `israeli-bank-scrapers` only in one place.

Because now, we're calling this also in the `configManager`, and it can create a mismatch.

Reply:
I will export it from the originalBudgetTracking index file so the ui code will not import it directly from the israeli-bank-scrapers library to give us more control